src/saveToCase.js

```diff
@@ -40,6 +42,10 @@ define([
         return mug ? mug.p.use_index : false;
     }
 
+    function attachmentCase(mug) {
```

Comment:
I see now that would be inconsistent with the other functions that do similar boolean checks.